docs/catalogue/cat_admin.md

```diff
@@ -47,3 +47,4 @@ procedure is to update.
 | 2.4 => 2.5  | CollectionEvents.coreVariables, Subcohorts.comorbidity, Subcohorts.counts. Add table SubcohortCounts. Download whole database, replace molgenis.csv and reupload in new Schema.                                                                                                                                                                                                                                                          | |
 | 2.5 => 2.6  | Edits to table SubcohortCounts: delete SubcohortCounts.year, SubcohortCounts.ageband, SubcohortCounts.gender, SubcohortCounts.N. Add SubcohortCounts.ageGroup, SubcohortCounts.N total, SubcohortCounts.N female, SubcohortCounts.N male. Download whole database, replace molgenis.csv and reupload in new Schema.                                                                                                                      | |
 | 2.6 => 2.7  | Add Network.type and CatalogueOntologies.NetworkTypes. To update: upload molgenis.csv and add 'h2020' and 'harmonization' to NetworkTypes                                                                                                                                                                                                                                                                                                | |
+| 2.7 => 2.8  | fix: Cohorts.designDescription columnType should be text. Just upload molgenis.csv to update.                                                                                                                                                                                                                                                                                                | |
```

Comment:
maybe its more of a change then a fix; "_Change Cohorts.designDescription columnType from STRING to TEXT. upload molgenis.csv to update_"